src/main/java/org/jabref/gui/groups/GroupTree.css

```diff
@@ -1,10 +1,10 @@
 .tree-table-view, .tree-table-row-cell {
-    -fx-background-color: #dadad8;
+    -fx-background-color: -fx-control-inner-background;
```

Comment:
I would introduce a new color `-jr-sidepane-background` here.

Reply:
OK, for a scheme that wants to set a different bg for the sidepane than in other controls this makes sense.